app/lib/filters/status.rb

```diff
@@ -3,13 +3,20 @@
 class Filters::Status < Filters::Base
   def apply
     return scope if statuses.empty?
-
     scope.where(status: statuses)
   end
 
   private
 
   def statuses
-    Array(params[:statuses]).reject(&:blank?)
+    Array(params[:statuses])
+      .reject(&:blank?)
+      .flat_map do |status|
+        if %w[initial_assessment assessment_in_progress].include?(status)
+          %w[assessment_in_progress]
```

Comment:
```suggestion
          %w[initial_assessment assessment_in_progress]
```

If the user selects the `assessment_in_progress` status, we want it to also include any existing applications with the `initial_assessment` status.

Reply:
Would that not just write them to a flat string of the two combined?

Reply:
Yes, it would, because we want to then filter the database like:

```rb
where(status: %w[initial_assessment assessment_in_progress])
```

Reply:
https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/a0932a42e8e956dab49dd3ad7220f681b56cc8a4/app/lib/filters/status.rb#L7

This is where we do the filter on the database. During this migration, we'll have applications in both the `initial_assessment` and `assessment_in_progress` status, which we want to consider the same, so if a user chooses to filter on "In progress" applications, we'll need to show them applications in both status.